vm/src/obj/objint.rs

```diff
@@ -653,6 +653,20 @@ impl PyInt {
     fn denominator(&self) -> usize {
         1
     }
+
+    #[pymethod]
+    /// Returns the number of ones 1 an int. When the number is < 0,
+    /// then it returns the number of ones of the absolute value.
+    fn bit_count(&self, vm: &VirtualMachine) -> PyResult {
+        Ok(vm.ctx.new_bigint(&BigInt::from(
+            self.value
+                .to_u32_digits()
+                .1
+                .iter()
+                .map(|n| n.count_ones())
+                .sum::<u32>(),
+        )))
+    }
```

Comment:
You could just return a u32 here:

```suggestion
    fn bit_count(&self) -> u32 {
        self.value
            .to_u32_digits()
            .1
            .iter()
            .map(|n| n.count_ones())
            .sum()
    }
```